Dockerfile

```diff
@@ -1,7 +1,8 @@
 FROM registry.fedoraproject.org/fedora:29
 
 LABEL maintainer="Factory 2 Team" \
-      description="A microservice triggered by specific message to tag a build."
+      description="A microservice triggered by specific message to tag a build." \
+      version="latest"
```

Comment:
I think this is ok, but we should probably investigate how to set this to a meaningful value.